docs/configuration/README.md

```diff
@@ -1,5 +1,10 @@
 # Configuration
 
+There are two options for configuration:
```

Comment:
Can we call out somewhere that both can be used and will be merged?  Also, if something is specified in both which takes precedence?

Reply:
Yes, I am not sure why I forgot to include this. Good call out 👍